config/config.go

```diff
@@ -1,3 +1,17 @@
+// Copyright 2017 Alejandro Sirgo Rica
+// Copyright 2019 Max Inden, Richard Hartmann, and The Prometheus Authors
```

Comment:
This should go in a NOTICE file. Also this project is outside of the Prometheus Authors, so it should just be Richard Hartmann for the current copyright.

Reply:
Will do!